studio.kx.carla.json

```diff
@@ -8,11 +8,12 @@
     "rename-icon": "carla",
     "finish-args": [
         "--share=ipc",
-        "--socket=x11",
         "--socket=wayland",
```

Comment:
```suggestion
```

I really meant to remove wayland too.

Reply:
this was already added in the initial PR as developer wanted Carla to have wayland access. Is this really problematic / have you tested it?
afaik this doesn't force wayland, only permits.
I just tested this in a wayland session, Carla running as flatpak with wayland access.
x42 plugins, LSP plugins, Surge, Vitalium and guitarix plugins run and display their UI just fine.
In which cases this cause problems?

Reply:
ok then